stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -70,19 +72,33 @@ public static void main(String[] args) {
 
   @Override
   public String getBenchDescription() {
-    return "This stress bench is designed for testing the reading throughput of Fuse-based "
-        + "POSIX API. To write data, run `bin/alluxio runClass alluxio.stress.cli.fuse."
-        + "fuseIOBench --operation Write`. To read data, run `bin/alluxio runClass "
-        + "alluxio.stress.cli.fuse.fuseIOBench --operation Read`. You can further adjust "
-        + "the parameters specified below. Note that the \"--operation\" is required, the "
-        + "\"--local-path\" can be a local filesystem path or a mounted fuse path, and test files "
-        + "need to be written first before reading.\nFor example, run `bin/alluxio runClass "
-        + "alluxio.stress.cli.fuse.fuseIOBench --operation Write --local-path /mnt/alluxio-fuse"
-        + "/FuseIOTest --num-files 100 --file-size 100m --threads 32` to write the test files "
-        + "first, then run `bin/alluxio runClass alluxio.stress.cli.fuse.fuseIOBench "
-        + "--operation Read --local-path /mnt/alluxio-fuse/FuseIOTest --num-files 100 "
-        + "--file-size 100m --threads 16 --warmup 15s --duration 30s` to test the reading "
-        + "throughput.\n";
+    return String.join("\n", ImmutableList.of(
+        "A stress bench for testing the reading throughput of Fuse-based POSIX API.",
+        "To run the test, data must be written first by executing \"Write\" operation, then "
+            + "run \"Read\" operation to test the reading throughput. Optionally one can run "
+            + "\"ListFile\" before \"Read\" to cache the metadata of the test files and eliminate "
+            + "the effect of metadata operations while getting the reading throughput data.",
+        "Note that \"--operation\" is required, the \"--local-path\" can be a local "
+            + "filesystem path or a mounted Fuse path, and if one wants to cache the metadata, "
+            + "set alluxio.user.metadata.cache.enabled=true when mounting the Alluxio Fuse.",
+        "",
+        "Example:",
+        "# The test data will be written to /mnt/alluxio-fuse/FuseIOTest",
+        "# Files will be evenly distributed into 10 directories, each contains 10 files of "
+            + "size 100 MB",
+        "# 32 threads will be used for writing the data, and 16 threads will be used for "
+            + "testing the reading throughput",
+        "# 5 seconds of warmup time and 30 seconds of actual reading test time",
+        "$ bin/alluxio runClass alluxio.stress.cli.fuse.fuseIOBench --operation Write \\",
+        "--local-path /mnt/alluxio-fuse/FuseIOTest --num-dirs 10 --num-files-per-dir 10 \\",
+        "--file-size 100m --threads 32",
```

Comment:
i remember we have a logics that num dirs should be bigger than thread num?

Reply:
Yes